patches/buildroot/0005-Bring-back-sysroot-fix-for-binutils.patch

```diff
@@ -0,0 +1,195 @@
+From 9afc3195d55b209fbf6668ed159198a930b9163b Mon Sep 17 00:00:00 2001
```

Comment:
Generally I'd recommend only adding the patch for the binutils version we're using; otherwise it'll get tripped up on rebase if the older binutils versions are dropped.

But that's a minor hiccup and here I think the value is that we could revert to an older binutils more easily if the 2.41 update causes problems.